src/tests/regression/ipset_scale.at

```diff
@@ -16,4 +37,7 @@ FWD_CHECK([--permanent --ipset foobar --add-entries-from-file ./entries], 0, [ig
 
 FWD_RELOAD()
 
+# We only ran a part of the test. Overall, mark the test as skipped.
```

Comment:
Why bother with the above if we're just going to skip here? Just call `SKIP_IF_FW_IN_CONTAINER_WITH_NFTABLES()` at the start of this test.

Reply:
The test might still do something interesting and testworthy with a reduced configuration.

Sure, it intentionally creates a large input (of mostly redundant input data). We cannot test *that* aspect. But otherwise, all steps with a smaller configuration should pass.

Skipping the test entirely, without running the parts that should pass, seems a wasted opportunity for coverage. As the hardest part was already done: to write the test.

Reply:
changed.